ffi-cdecl/mxcfb_remarkable_decl.c

```diff
@@ -0,0 +1,45 @@
+// standard Linux framebuffer headers
```

Comment:
And then update & refresh that accordingly ;).

Reply:
Done